haystack/components/generators/chat/fallback.py

```diff
@@ -47,22 +48,27 @@ class FallbackChatGenerator:
     - Any other exception
     """
 
-    def __init__(self, chat_generators: list[ChatGenerator]):
+    def __init__(self, chat_generators: list[ChatGenerator], streaming_callback: Optional[StreamingCallbackT] = None):
```

Comment:
Let's pre-emptively make any new init params keyword only in case we add more in the future
```suggestion
    def __init__(self, chat_generators: list[ChatGenerator], *, streaming_callback: Optional[StreamingCallbackT] = None):
```